src/java.base/share/classes/java/net/NetworkInterface.java

```diff
@@ -97,6 +114,11 @@ public String getName() {
      * {@link NetPermission}("getNetworkInformation") permission, then all
      * InetAddresses are returned.
      *
+     * @implNote
+     * The returned array contains all or a subset of the InetAddresses that were
+     * bound to the interface at the time the {@linkplain #getNetworkInterfaces()
+     * interface configuration was read}
```

Comment:
I think you have a typo here in that this method returns an Enumeration rather than an array.